model/distro/distro.go

```diff
@@ -55,6 +55,21 @@ type Distro struct {
 	IcecreamSettings      IcecreamSettings      `bson:"icecream_settings" json:"icecream_settings" yaml:"icecream_settings"`
 }
 
+type DistroData struct {
+	Distro              Distro                   `bson:",inline"`
```

Comment:
You can embed the distro in this struct too to match the bson, unless there's a reason you didn't